### PR TITLE
Update issue templates to add Feature Enhancement

### PR DESCRIPTION
**Pull request by kiwik**
1. Add a new template for feature enhancement for Openlab
2. Add default label for each template

Closes: theopenlab/openlab#203

### CODE REVIEW

**Review by liusheng — DISMISSED** [review]
lgtm

**Comment by kiwik** [comment]
And I have update OpenLab document website, see here https://docs.openlabtesting.org/publications/contributors/development-guide#working-on-feature-and-bug to add feature related workflow

**Review by mrhillsman — CHANGES_REQUESTED** [review]
Let's add hogepodge and stmcginnis as assignees for test requests; maybe that will speed them up

**Review by huangtianhua — DISMISSED** [review]
lgtm，thanks

**Comment by kiwik** [comment]
@mrhillsman have added them into reviewer list

**Comment by kiwik** [comment]
@mrhillsman address your comment

File: .github/ISSUE_TEMPLATE/report-bug.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 name: Report Bug
 about: Report bug to OpenLab.
+title: ''
+labels: bug
+assignees: ''
 
 ---
 
-Let's make things better!
+Thank you for reporting bug!
```
Comment (stmcginnis) [inline .github/ISSUE_TEMPLATE/report-bug.md]: "reporting a bug" or "reporting this bug" would be more grammatically correct, but I don't think it's a big deal.

**Review by mrhillsman — APPROVED** [review]

**Review by stmcginnis — APPROVED** [review]
LGTM

**Pull request merged**